src/transport/GroupSession.h

```diff
@@ -45,7 +45,7 @@ class IncomingGroupSession : public Session, public ReferenceCounted<IncomingGro
 
     Session::SessionType GetSessionType() const override { return Session::SessionType::kGroupIncoming; }
 #if CHIP_PROGRESS_LOGGING
-    const char * GetSessionTypeString() const override { return "incoming group"; };
+    const char * GetSessionTypeString() const override { return "G"; };
 #endif
```

Comment:
Logging code could have SessionType --> string mapping, rather than this string getter.

Reply:
See [this](https://github.com/project-chip/connectedhomeip/pull/22465#discussion_r969875486).